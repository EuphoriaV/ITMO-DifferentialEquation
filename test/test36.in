3.27
((x + (((9.62 - x) + x) * 7.17)) - ((x + (((x + 5.1) - 6.22) - (3.38 * 8.6))) * ((1.59 * x) * (x * 3.08))))
1.0
1.6