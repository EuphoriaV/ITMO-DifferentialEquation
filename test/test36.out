3.6665699